templates/index.html

```diff
@@ -1,129 +1,181 @@
+				</div>
+			</div>
+
+			<div class="col-md-2">
+				<div class='icon' id='classic' href=#>
+					<INPUT type="image" src="{{url_for('static',filename = 'images/classic.png')}}"
+					onclick="updateData()" width="85%">
+				</div>
+			</div>
+
+			<div class="col-md-2">
+				<div class='icon' id='hiphop' href=#>
+					<INPUT type="image" src="{{url_for('static',filename = 'images/hiphop.png')}}"
+					onclick="updateData()" width="85%">
+				</div>
+			</div>
+
+			<div class="col-md-2">
+				<div class='icon' id='metal' href=#>
+					<INPUT type="image" src="{{url_for('static',filename = 'images/metal.png')}}"
+					onclick="updateData()" width="85%">
+				</div>
+			</div>
+
+		</div>
+		<div class= "row">
+		<div class= "col-md-11" id="mapCanvas">
+			<div> <!-- div and svg to hold the map. -->
+				<svg id = "statesvg" viewBox = "0 0 960 630" preserveAspectRatio="xMidYMid meet"></svg>
+			</div>
+			<div id="tooltip"></div><!-- div to hold tooltip. -->
+		</div>
+		<div class = col-md-1 id="legend"></div><!-- div to hold legend. -->
+		</div>
 
 </div>
 </body>
 
 
-<!-- SOURCE: https://gist.github.com/NPashaP/a74faf20b492ad377312 -->
-<script src="{{url_for('static',filename = 'uStates.js')}}"></script> <!-- creates uStates. -->
-<script src="http://d3js.org/d3.v3.min.js"></script>
+<script src = "https://cdnjs.cloudflare.com/ajax/libs/d3/4.2.2/d3.min.js"></script><!-- D3 graphing library -->
+<script src="https://d3js.org/d3-scale-chromatic.v1.min.js"></script>
+<script src = "https://d3js.org/topojson.v1.min.js"></script><!-- topojson library -->
+<script src="{{url_for('static',filename = 'uStates.js')}}"></script><!-- creates uStates. -->
 <script>
-	function tooltipHtml(n, d){	/* function to create html content string in tooltip div. */
-		return "<h4>"+n+"</h4><table>"+
-			"<tr><td>Low</td><td>"+(d.low)+"</td></tr>"+
-			"<tr><td>Average</td><td>"+(d.avg)+"</td></tr>"+
-			"<tr><td>High</td><td>"+(d.high)+"</td></tr>"+
-			"<small>(click to zoom)</small>"+
-			"</table>";
-	}
-
-	var classicData ={};	/* Sample random data. */
-	["HI", "AK", "FL", "SC", "GA", "AL", "NC", "TN", "RI", "CT", "MA",
-	"ME", "NH", "VT", "NY", "NJ", "PA", "DE", "MD", "WV", "KY", "OH",
-	"MI", "WY", "MT", "ID", "WA", "DC", "TX", "CA", "AZ", "NV", "UT",
-	"CO", "NM", "OR", "ND", "SD", "NE", "IA", "MS", "IN", "IL", "MN",
-	"WI", "MO", "AR", "OK", "KS", "LS", "VA"]
-		.forEach(function(d){
-			var low=Math.round(100*Math.random()),
-				mid=Math.round(100*Math.random()),
-				high=Math.round(100*Math.random());
-			classicData[d]={low:d3.min([low,mid,high]), high:d3.max([low,mid,high]),
-					avg:Math.round((low+mid+high)/3), color:d3.interpolate("#b51062", "white")(low/100)};
+
+	uStates.draw("most_pop")
+
+	//Update data section (Called from the onclick)
+	function updateData() {
+		d3.select('#logo')
+		.on('click', function() {
+	 		d3.selectAll("svg > *").remove();
+			uStates.draw("most_pop");
+		});
+
+		d3.select('#pop')
```

Comment:
You can have this event fire on a `class` instead of an `id`, so it gets fired for *all* genres, and you can then have:
```javascript
d3.select('.genre-icon')
  .on('click', function() {
    const genre_id = d3.select(this).attr('id');
    d3.selectAll("svg > *").remove();
    uStates.draw(genre_id);
  });
```